### PR TITLE
Translation updated for 3.3.x

### PR DESCRIPTION
**Pull request by TimVNL**

### CODE REVIEW

File: messages.php [hunk 1]
```diff
@@ -126,6 +126,7 @@
     /*
      * 2.1.2
      */
-    "field_required"        => "Veld is verplicht.",
+    "popup_blocked"         => "Uw browser blokkeert pop-upvensters. Schakel pop-ups en omleidingen op deze website in om de voorbeeldfunctie te gebruiken. ",
```
Comment (jshah4517) [inline messages.php:129]: This file is missing the 2.3.0 to 2.5.0 translations

File: permission.php [hunk 2]
```diff
@@ -95,4 +95,9 @@
     // Core
     "logfiles"          => "Beheer logbestanden",
```
Comment (jshah4517) [inline permission.php:96]: This file is missing 2.4.0 to 3.0.0 translations

File: validation.php [hunk 3]
```diff
@@ -91,13 +86,88 @@
...
+
     /*
      * 2.2.0
      */
-    "required_with_translation" => "De :translation vertaling voor het :attribute veld is verplicht wanneer :values is ingesteld.",
-    "max_translation"           => "De :translation vertaling voor het :attribute veld mag niet langer dan :max tekens zijn.",
-    "unique_translation"        => "De :translation vertaling voor het :attribute veld is reeds ingebruik.",
+    "required_with_translation" => "The :translation translation for the :attribute field is required when :values is present.",
+    "max_translation"           => "The :translation translation for the :attribute field may not be greater than :max characters.",
+    "unique_translation"        => "The :translation translation for the :attribute field has already been taken.",
```
Comment (jshah4517) [inline validation.php:95]: Need to revert the translations for the strings that are still there.

File: validation.php [hunk 4]
```diff
@@ -69,17 +69,12 @@
...
-    "json"                      => "De :attribute moet een geldige JSON zijn.",
-    "user_password_strength"    => "De :attribute moet :user_password_strength zijn.",
-    "operator_password_strength" => "De :attribute moet contain: :operator_password_strength zijn.",
-    "twig_lint"                 => "De :attribute moet geldige twig syntax zijn, zie http://twig.sensiolabs.org/doc/templates.html",
+    "unique"                    => "The :attribute has already been taken.",
+    "url"                       => "The :attribute format is invalid.",
+    "template_exists"           => "The selected :attribute is invalid.",
+    "is_valid_captcha"          => "The captcha code entered was invalid or incorrect, please try again.",
+    "user_password_strength"    => "The :attribute must contain: :user_password_strength.",
+    "operator_password_strength" => "The :attribute must contain: :operator_password_strength.",
```
Comment (jshah4517) [inline validation.php:77]: Need to revert the translations for the strings that are still there.

File: actions.php [hunk 5]
```diff
@@ -13,15 +13,15 @@
...
-    
+
     /*
      * 2.0.3
      */
-    "update_custom_field"       => "Update custom field van ticket",
+    "update_custom_field"       => "Update aangepast veld van ticket",
 
     /*
      * 2.2.0
```
Comment (jshah4517) [inline actions.php:27]: This file is missing the 2.3.0 to 2.6.0 translations

**Review by jshah4517 — COMMENTED** [review]
Hi Tim,

Thanks for your effort in updating the translations and contributing them, we really appreciate it!

I have done a review and noticed a small number of missing translations and validation.php has some cases where they have been overwritten when copying over the latest strings. Would you be happy to check over these and see if you are able to translate the missing strings?

**Comment by TimVNL** [comment]
Hello Jay, 
Thanks for your Feedback :)

I will add the missing translation strings and complete the translation I will make a new pull request once it's done.

File: actions.php (hunk 5 above)
Comment (TimVNL) [reply actions.php:27]: added the missing strings and finalized translation for this file

File: permission.php (hunk 2 above)
Comment (TimVNL) [reply permission.php:96]: Added missing strings and completed translation of this file

**Comment by TimVNL** [comment]
Hey Jay, 

I am almost done with the translation but supportpol gives me a 500 ERROR when the language is active in the admin and when I go to the language settings I get an ERROR how can i find which line is causing this?

**Comment by TimVNL** [comment]
Hello Jay,

The translation is complete :) 
Could you take a look if it can be merged now?

**Comment by jshah4517** [comment]
Hi Tim,

Thanks again! I did a deeper review and found a few more things, I pushed it directly to your fork - https://github.com/TimVNL/supportpal-language-files/commit/204fc523ef8eba386df59d0b2a86dbb65c584a19

Would you like to translate some of these too or should I go ahead and merge?

**Comment by TimVNL** [comment]
Hello Jay,

I will take a look at the missing translations :)
Thanks for the deeper review.

**Comment by jshah4517** [comment]
Thank you again 👍🏽

**Pull request merged**